tests/csm/rest/test_iam_users.py

```diff
@@ -386,7 +409,1460 @@ def test_35607(self):
         self.log.info("TODO Verify Response : %s", resp)
         self.log.info("[END]Creating IAM user with basic parameters")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+    
+    ###
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-35608')
+    def test_35608(self):
+        """
+        Test that user can't create IAM user with duplicate parameters.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Testing IAM user with duplicate parameters")
+        self.log.info("Creating IAM user payload.")
+        user_id, display_name, email = self.get_IAM_user_payload("email")
+        payload = {"uid": user_id, "display_name": display_name, "email": email}
+        self.log.info("payload :  %s", payload)
+
+        self.log.info("Creating IAM user.")
+        response = self.csm_obj.create_iam_user_rgw(payload)
+        assert response.status_code == HTTPStatus.CREATED, "Status code check failed for user creation"
+        resp = response.json()
+        self.log.info("Performing POST API to Create IAM User with same uid as above.")
+        payload = {"uid": user_id, "display_name": display_name}
+        self.log.info("payload :  %s", payload)
+        resp_new = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify user exist error.")
+        assert resp_new.status_code == HTTPStatus.CONFLICT, "Check failed for duplicate user creation"
+
+        self.log.info("Perform API to Create IAM User with same email Id as above.")
+        user_id2, display_name2 = self.get_IAM_user_payload()
+        payload = {"uid": user_id2, "display_name": display_name2, "email": email}
+        self.log.info("payload :  %s", payload)
+        resp_new = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify status email exist error.")
+        assert resp_new.status_code == HTTPStatus.CONFLICT, "Check failed for duplicate user creation"
+
+        self.log.info("Perform API to Create IAM User with already existing user Access Keys.")
+        user_id3, display_name3, email3 = self.get_IAM_user_payload("email")
+        payload = {"uid": user_id3, "display_name": display_name3, "email": email3,
+                   "access_key": resp["keys"][0]["access_key"], "secret_key": resp["keys"][0]["secret_key"]}
+
+        self.log.info("payload:  %s", payload)
+        self.log.info("Verify keys_exist error)")
+        resp_new = self.csm_obj.create_iam_user_rgw(payload)
+        assert resp_new.status_code == HTTPStatus.CONFLICT, "Check failed for duplicate user creation"
+        self.log.info("[END] Testing with duplicate uid and email")
```

Comment:
Suggestion added completed statement as it is easier to debug
        self.log.info("##### Test ended -  %s #####", test_case_name)

Reply:
Resolved @kachhwahadivya